templates/publisher/listing.html

```diff
@@ -517,12 +525,13 @@
         document.getElementById('license-custom-input')
       ]
     }, {
+      snap_name: {{ snap_name|tojson }},
       title: {{ snap_title|tojson }},
       summary: {{ summary|tojson }},
       description: {{ description|tojson }},
       website: {{ website|tojson }},
       contact: {{ contact|tojson }},
-      categories: "{% if snap_categories.categories[0] %}{% for category in snap_categories.categories %}{% if loop.index > 1 %}, {% endif %}{{ category.name }}{% endfor %}{% endif %}",
+      categories: {{ snap_categories.categories|tojson }},
```

Comment:
Is it supposed to support categories that are not merged yet?
I believe that branch uses `categories` in the context of template.